en/django_installation/instructions.md

```diff
@@ -104,6 +104,26 @@ $ python3 -m venv myvenv
 
 <!--endsec-->
 
+<!--sec data-title="Virtual Environment: RunCode" data-id="virtualenv-runcode" -->
```

Comment:
If this is added into an existing resource to show an alternative methods when using RunCode, you will also need to add further up that there are two ways to do this. The first, if you have followed the installation steps (followed by existing instructions), then another sub heading for if you have installed RunCode followed by these new instructions.